pkg/upload/react-native-ios.go

```diff
@@ -68,123 +71,118 @@ func ProcessReactNativeIos(options options.CLI, endpoint string, logger log.Logg
 						logger.Warn(err.Error())
 					}
 				}
-
-				if iosOptions.Ios.Scheme != "" {
-					buildSettings, err = ios.GetXcodeBuildSettings(iosOptions.Ios.XcodeProject, iosOptions.Ios.Scheme, "")
-					if err != nil {
-						logger.Warn(err.Error())
-					}
-				}
 			} else {
 				return fmt.Errorf("Could not find an Xcode project file, please specify the path by using --xcode-proj-path")
 			}
 
+			logger.Debug(fmt.Sprintf("Found Xcode Scheme: %s", iosOptions.Ios.Scheme))
+
+			xcodeArchivePath, err = ios.GetLatestXcodeArchiveForScheme(iosOptions.Ios.Scheme)
+
+			if err != nil {
+				return fmt.Errorf("Error locating latest xcarchive: %w", err)
```

Comment:
Shall we say "...from Xcode project (scheme XXX), please specify the archive path directly using --archive-path"